atc/db/worker_resource_cache.go

```diff
@@ -19,10 +18,14 @@ type UsedWorkerResourceCache struct {
 var ErrWorkerBaseResourceTypeDisappeared = errors.New("worker base resource type disappeared")
 
 func (workerResourceCache WorkerResourceCache) FindOrCreate(tx Tx) (*UsedWorkerResourceCache, error) {
```

Comment:
Maybe it's better to just have `sourceWorkerName` as an argument to `FindOrCreate`, rather than adding it to the struct when its never used for `Find`

Reply:
Good suggestion.